packages/flutter/lib/src/material/slider.dart

```diff
@@ -463,6 +525,25 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
       semanticFormatterCallback: widget.semanticFormatterCallback,
     );
   }
+
+  Widget _buildCupertinoSlider(BuildContext context) {
+    // The render box of a slider has a fixed height by takes up the available
+    // width. Wrapping the cupertino slider in this manner will help ensure
```

Comment:
[CupertinoSlider]. Might be others in this comment.

Reply:
Done